src/manifest.json

```diff
@@ -23,4 +23,4 @@
       "run_at":"document_idle"
```

Comment:
![badge](https://img.shields.io/badge/review-suggest-brightgreen.svg)

If it is document_idle, sometimes it is not applied properly, so document_end is considered better.

```suggestion
      "run_at":"document_end"
```